src/plugins/internal-calling-user.js

```diff
@@ -5,6 +5,7 @@ const { get, set } = require('lodash');
 
 const { logger } = require('../logger');
 const idmConnector = require('../lib/connectors/idm');
+const User = require('../lib/models/user');
 
 const validateUserIsInternal = user => {
   if (user.application !== 'water_admin') {
```

Comment:
I think I'm not 100% sure what this is doing. Might it be worth adding a comment?

It looks like it's a function to check whether the user is internal/admin, but the if statement is then checking if the user is NOT admin. Bit confused, but I think it's likely that I'm just not reading it correctly :)